src/koa-module-specifier-transform.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright (c) 2019 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import getStream from 'get-stream';
+import * as Koa from 'koa';
+import {Stream} from 'stream';
+import {transformHTMLString} from './transform-html';
+import {transformJavaScriptModuleString} from './transform-javascript-module';
+
+export type TransformSpecifierFunction = (baseURL: string, specifier: string) =>
+    string;
+
+export const koaModuleSpecifierTransform = (transformSpecifier:
```

Comment:
not sure the `koa` prefix is needed here

Reply:
Dropped the koa prefix.